yb-voyager/src/srcdb/postgres.go

```diff
@@ -469,3 +469,32 @@ func generateSSLQueryStringIfNotExists(s *Source) string {
 func (pg *PostgreSQL) GetServers() []string {
 	return []string{pg.source.Host}
 }
+
+func (pg *PostgreSQL) GetChildPartitions(tableName *sqlname.SourceName) []*sqlname.SourceName {
+	childPartitions := make([]*sqlname.SourceName, 0)
+	query := fmt.Sprintf(`SELECT
+    nmsp_child.nspname  AS child_schema,
+    child.relname       AS child
+FROM pg_inherits
```

Comment:
Are we sure pg_inherits table ONLY includes partition-type relationships?

Reply:
Yes, It stores the inheritance related information of the tables - 

> The catalog pg_inherits records information about table inheritance hierarchies. There is one entry for each direct child table in the database. (Indirect inheritance can be determined by following chains of entries.)

Reply:
@priyanshi-yb looks like "inheritance" in PG is a lot more than just partitions.. https://www.postgresql.org/docs/current/tutorial-inheritance.html

I think we should get more clarity here. Either we find a way to perform the above logic ONLY for partition-type inheritance, or we confirm that even in case of other inheritance relationships, this logic still is correct..

Reply:
Yes sure, let me understand this more and figure out the proper thing for our use case.

Reply:
I have verified that this Query is still correct in the case of other Inheritance relationships with the example given in the above link so we are good with this.
On a side note, YugabyteDB doesn't support INHERITS at this moment ([GH issue](https://github.com/yugabyte/yugabyte-db/issues/1129)).